Contributing.md

```diff
@@ -0,0 +1,39 @@
+# Contribution Guide
+
+Please follow the instructions on this guide when making contributions to the JCT Discord Bot.
+
+## First Time Contributor
+
+The first time you contribute to this project, feel free to add your name under the list of contributors in the [README.md](https://github.com/DenverCoder1/jct-discord-bot/blob/main/README.md) file. You can include this change along with the pull request you make.
```

Comment:
This can be automated using GitHub actions

Reply:
Cool, didn't know that. We'll remove it once implemented?